core/ExceptionHandler.php

```diff
@@ -113,11 +113,13 @@ public static function replaceSensitiveValues(string $message): string
         $dbConfig = Db::getDatabaseConfig();
 
         $valuesToReplace = [
-            Piwik::getCurrentUserTokenAuth() => 'tokenauth',
-            SettingsPiwik::getSalt()         => 'generalSalt',
-            $dbConfig['username']            => 'dbuser',
-            $dbConfig['password']            => 'dbpass',
+            'tokenauth'   => Piwik::getCurrentUserTokenAuth() ?: '',
```

Comment:
We wouldn't need the `?: ''` addition with the array_filter approach as that removes all falsey values, including nulls, zeros, empty strings etc.

Reply:
True, but it doesn't hurt anything either. I just removed it.